docs/UsersGuide/source/Prerequisites.rst

```diff
@@ -0,0 +1,25 @@
+.. _Prerequisites:
+
+**************************************************
+Prerequisites for Working with the HAFS Repository
```

Comment:
Is HPC-Stack/spack-stack required? For UFS repositories, users generally need more than just Git.

Reply:
I believe so. Basically, HAFS needs majority of the modules/libraries listed in, e.g., https://github.com/hafs-community/HAFS/blob/develop/modulefiles/hafs.orion.lua